### DISCUSSION TITLE
2.0.1 and previous experimental flags (extendedSpacingScale)

### DISCUSSION THREAD

**Discussion by Delkathes** [tailwindlabs/tailwindcss#2801] [Help]
Hi guys,

Just updated to 2.0.1 following the upgrade guide. But I can't find anything about the experimental "extendedSpacingScale" feature. All of my `pb-X/X` don't work anymore. And when searching for Experimental on 2.0.1 docs, u'll get a 404.

https://tailwindcss.com/docs/release-notes#experimental => 404
https://tailwindcss.com/docs/release-notes#experimental-features => 404

Does someone know how to bring that extendedSpacingScale thing back ?

Thanks

**Comment by MartinHughes-BPC** [comment]
There's something about this (I think) in the Tailwind UI docs. Either use the new syntax or add fractional values to the padding scale.

https://tailwindui.com/changes-for-v2
